app/models/drop_one_member_topics.rb

```diff
@@ -0,0 +1,20 @@
+class DropOneMemberTopics < DeadlineType
+
+def email_list(assignment_id)
+  assignment = Assignment.find(assignment_id)
+  drop_one_member_topics if (assignment.team_assignment?)
+  emails =[]
+  emails     
+end
+
+def drop_one_member_topics
+ teams = TeamsUser.all.group(:team_id).count(:team_id)
```

Comment:
This is not correct. All the teams with only 1 member will be dropped, which is definitely not correct. There are many assignments in Expertiza require individual submissions. Instead, you should find the teams _in this assignment_ which have only one member.